### PR TITLE
Upgrade to Molecule v2 and Ansible minimum version 2.4.5.0

### LINKED ISSUE #5: Upgrade Molecule version (and Ansible minimum version)
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Checked that your issue isn't already filled: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aidealista
    * Checked that there is not already provided the described functionality

### Description

It has become a bit difficult to test the role with Molecule 1.25, so upgrade the Molecule version (and therefore, Ansible minimum version).

### PR DESCRIPTION
**Pull request by jnogol**
- Fixes #5

### CODE REVIEW

**Review by jperera — APPROVED** [review]

File: molecule/default/molecule.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  name: galaxy
+driver:
+  name: docker
+lint:
+  name: yamllint
+  enabled: False
+
+platforms:
+  - name: opengrok-${MOLECULE_DISTRO:-stretch}
+    image: debian:${MOLECULE_DISTRO:-stretch}
```
Comment (dortegau) [inline molecule/default/molecule.yml]: I would prefer to use our java image based on Debian

File: molecule/default/verify.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    goss_version: v0.3.6
+    goss_arch: amd64
+    goss_dst: /usr/local/bin/goss
+    goss_url: "https://github.com/aelsabbahy/goss/releases/download/{{ goss_version }}/goss-linux-{{ goss_arch }}"
+    goss_test_directory: /tmp
+    goss_format: documentation
+
+  vars_files:
+    - ../../defaults/main.yml
+    - ./group_vars/airflow/main.yml
```
Comment (dortegau) [inline molecule/default/verify.yml]: C&P 😀

File: tasks/main.yml [hunk 3]
```diff
@@ -1,22 +1,27 @@
 ---
 - name: OpenGrok | Install
-  include: install.yml
+  import_tasks: install.yml
   tags:
     - install
```
Comment (dortegau) [inline tasks/main.yml]: The generic tags like `install` should be removed, shouldn't be?

Comment (jnogol) [reply tasks/main.yml]: done

File: molecule/default/verify.yml (hunk 2 above)
Comment (jnogol) [reply molecule/default/verify.yml]: yup XD. Fixed.

File: molecule/default/molecule.yml (hunk 1 above)
Comment (jnogol) [reply molecule/default/molecule.yml]: done

**Comment by dortegau** [comment]
Maybe this section should be updated: https://github.com/idealista/opengrok-role#prerequisities (talks about Vagrant, landrush, etc.)

**Comment by jnogol** [comment]
> Maybe this section should be updated: https://github.com/idealista/opengrok-role#prerequisities (talks about Vagrant, landrush, etc.)

Done :)

**Review by dortegau — APPROVED** [review]

**Review by sorobon — APPROVED** [review]

**Pull request merged**